docs/rules/accessor-pairs.md

```diff
@@ -0,0 +1,147 @@
+# Disallow setters without getters in objects (accessor-pairs)
+
+It's a common mistake in JavaScript to create an object with just setter but never have a getter defined for it.
+Here are some examples:
+
+```js
+// Bad
+var o = {
+    set a(value) {
+        val = value;
+    }
+};
+
+// Good
+var o = {
+    set a(value) {
+        val = value;
+    },
+    get a(value) {
+        return val;
+    }
+};
+
+```
+
+This rule warns if setters are defined without getters. Using an option `getWithNoSet`, it will warn if you have a getter without a setter also.
+
+## Rule Details
+
+This rule enforces a style where it requires to have a getter for every object which has a setter defined. By activating the below option it enforces vice-versa behaviour also.
+
+### Options
+
+`getWithNoSet` set to `true` would also check for getters without setters.
```

Comment:
`setWithoutGet` option should be exposed to the user?
If yes, what if user sets both option to false then basically this rule would be turned off. Is that what we want to do?